tests/bundle_poller_test.rs

```diff
@@ -20,7 +20,8 @@ mod tests {
         let config = BuilderConfig {
             host_chain_id: 17000,
             ru_chain_id: 17001,
-            host_rpc_url: "http://rpc.holesky.signet.sh".into(),
+            host_rpc_url: "host-rpc.example.com".into(),
+            ru_rpc_url: "ru-rpc.example.com".into(),
```

Comment:
These RPC URLs need to be set to proper values, they'll fail with these example ones. They're not being run right now in the typical test suite, though, which is why they're passing CI.

Reply:
Discarding my own comment - these values aren't used anymore like I thought they were. Will approve to unblock now.